### PR TITLE
feat: carousell downloads google analytics

### PR DESCRIPTION
**Pull request by osslgtm**
## Summary

Adding google analytics event for white paper download

## Changes

Carousel

## Issues

[Ticket](https://www.pivotaltracker.com/story/show/183309902)

### CODE REVIEW

File: src/components/Carousel/Carousel.tsx [hunk 1]
```diff
@@ -51,6 +53,17 @@ export const Carousel: FunctionComponent<CarouselProps> = ({ slides }) => {
...
+          if (!buttonDownload) {
+            return;
+          }
+          gaEvent({
+            category: GaCategory.FILE_DOWNLOAD,
+            action: GaAction.CAROUSEL_DOWNLOAD,
+            label: buttonDownload.file,
+          });
+          window.open(buttonDownload.file, "_blank", "noopener,noreferrer");
+        };
```
Comment (cavacado) [inline src/components/Carousel/Carousel.tsx:62]: did u check in the analytics console after event fired?

Comment (osslgtm) [reply src/components/Carousel/Carousel.tsx:62]: yeap, the events are showing up as events under "file_download" on the TT Dev

might need to figure out how to filter by labels on the dashboard

Comment (simboonlong) [reply src/components/Carousel/Carousel.tsx:62]: how about keeping the prev href and onClick fire directly GA tracking instead? so u:
- avoid the unnecessary [conditional check](https://github.com/TradeTrust/tradetrust-website/blob/ecfd2fbcc017ddb9e9b3e109d81cebe2fe48ed43/src/components/Carousel/Carousel.tsx#L57-L59) and [manual open new tab](https://github.com/TradeTrust/tradetrust-website/blob/ecfd2fbcc017ddb9e9b3e109d81cebe2fe48ed43/src/components/Carousel/Carousel.tsx#L65).
- adhere to the principles of unobtrusive js. (sorry old school thinking, technically entire app is js...)

if you want to have a new tab approach, use `target="_blank"` directly? let users ownself download themselves in the tab pdf view.

Comment (osslgtm) [reply src/components/Carousel/Carousel.tsx:62]: yeap fixed

**Review by simboonlong — APPROVED** [review]

**Pull request merged**